tests/support/helpers.py

```diff
@@ -153,10 +153,51 @@ def setUp(self, *args, **kwargs):
 
     # We're simply decorating functions
     @functools.wraps(caller)
-    def wrap(cls):
+    def wrap(cls, *args, **kwargs):
```

Comment:
Added `*args, **kwargs` because of `require_system_grains` usage?

Reply:
yes, if multiple decorators are used this would fail depending on the order. For example

```
@with_tempfile
@slowTest
def foo(): pass
```

Would fail because the method returned by `slowTest` isn't expecting any arguments or keywords. This change resolves that problem for slowTest, destrictiveTest, and expensiveTest.